src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpMessageHandlerIntegration.cs

```diff
@@ -22,7 +22,7 @@ public static class HttpMessageHandlerIntegration
         private const string HttpClientHandlerTypeName = "HttpClientHandler";
 
         private const string HttpMessageHandler = SystemNetHttp + "." + HttpMessageHandlerTypeName;
-        private const string HttpClientHandler = SystemNetHttp + "." + HttpClientHandlerTypeName;
+        private const string HttpClientHandlerConst = SystemNetHttp + "." + HttpClientHandlerTypeName;
```

Comment:
nit: should we rename both of these to `FooTypeFullName` to match the `FooTypeName` above? If I understand correctly, the difference is that these include the namespace with the name.

Reply:
Oh, wait, you added `Const` because there's now a setter property below.